app-compat-travis.yml

```diff
@@ -0,0 +1,22 @@
+# Test compatibility with applications written in ksh
+sudo: required
+
+services:
+    - docker
+
+matrix:
+    include:
+        - env:
+            - $PROJECT_NAME=kshdb
```

Comment:
Is the leading dollar-sign supposed to be there?

Reply:
Fixed!